lib/frontend/get-jira-configuration.js

```diff
@@ -52,6 +53,7 @@ module.exports = async (req, res, next) => {
       connections,
       failedConnections,
       hasConnections: connections.length > 0 || failedConnections.length > 0,
+      maintenanceMode: process.env.MAINTENANCE_MODE === 'true' || false,
```

Comment:
We can probably just do a check if _anything_ is set for maintenance mode instead with `maintenanceMode: !!process.env.MAINTENANCE_MODE` since the data being set on the flag isn't important.

Reply:
I did this on purpose to avoid misunderstandings (like when someone sets the env var to `MAINTENANCE_MODE=false`, this would still evaluate to `true` if I don't explicitly compare to the string "true".

Reply:
`|| false` - is it also to improve readability?

Reply:
ha, that's a good point, I didn't even think of that

Reply:
yeah, I'm a big fan of code that's a bit more verbose but leaves little to your imagination :)